mappings/net/minecraft/container/ContainerFactory.mapping

```diff
@@ -1,5 +1,5 @@
 CLASS net/minecraft/class_1270 net/minecraft/container/ContainerFactory
-	METHOD createMenu (ILnet/minecraft/class_1661;Lnet/minecraft/class_1657;)Lnet/minecraft/class_1703;
+	METHOD createMenu createMenu (ILnet/minecraft/class_1661;Lnet/minecraft/class_1657;)Lnet/minecraft/class_1703;
```

Comment:
```suggestion
	METHOD createMenu (ILnet/minecraft/class_1661;Lnet/minecraft/class_1657;)Lnet/minecraft/class_1703;
```
Unfortunatley please don't mark things as deobfuscated. you know, old enigma sucks sometimes.